tests/handlebars-basics/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,72 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`basic-handlebars.hbs 1`] = `
```

Comment:
why this deleted.

Reply:
Because it was basically empty.

This file contained two examples:
´´´
<script id="entry-template" type="text/x-handlebars-template">		
   <div class="entry">		
     <h1>{{title}}</h1>		
     <div class="body">		
       {{body}}		
     </div>		
   </div>		
 </script>
´´´	
and
´´´
  <div class="{{hello}} {{world}}"></div>
´´´

The first one is extracted to its own folder. I felt the second one is two basic to be relevant. Class attributes are tested in depth in test/handlebars-mustache-statement folder. Do you want me to revert the file deletion?

Reply:
Last time I try to update `glimmer` package, this file did report an error, not sure what error, better keep it.

Reply:
@fisker This PR is a solution exactly for that problem. It's okay.